packages/dd-trace/src/debugger/devtools_client/source-maps.js

```diff
@@ -35,14 +34,7 @@ const self = module.exports = {
   }
 }
 
-// The version check inside this function is to guard against a bug Node.js version 18, in which calls to `setTimeout`
-// might throw an uncatchable error from within `AsyncLocalStorage._propagate` with the following error message:
-//
-//     TypeError: Cannot read properties of undefined (reading 'Symbol(kResourceStore)')
-//
-// Source: https://github.com/nodejs/node/blob/v18.20.6/lib/async_hooks.js#L312
 function cacheIt (key, value) {
-  if (NODE_MAJOR < 20) return value
```

Comment:
same comment: if v5 still supports node18, we either still need this check or the code from master and v5 will diverge, making release harder. Let me know if I'm missing something

Reply:
This one is fine as is: the cache was deactivated because of a Node.js issue that was actually independent. It is now fixed in Node.js directly and deactivating did not fix the issue in the end. Thus, activating this is actually good.